File: src/CodeSwifterStarter.Application/DataEngine/DataEngine/PagingInfo.cs
namespace CodeSwifterStarter.Application.DataEngine.DataEngine
{
    public class PagingInfo
    {
        public int PageIndex { get; set; }
        public int PageSize { get; set; }
        public int TotalItems { get; set; }
        public bool PagingEnabled { get; set; }
    }
}

File: src/CodeSwifterStarter.Web.Api/ClientApp/src/app/models/data/common/paging-info.ts
export interface IPagingInfo {
  pageIndex: number;
  pageSize: number;
  totalItems: number;
  pagingEnabled: boolean;
}

export class PagingInfo implements IPagingInfo {
  pageIndex: number;
  pageSize: number;
  totalItems: number;
  pagingEnabled: boolean;


  public constructor(init?: Partial<PagingInfo>) {
    (Object as any).assign(this, init);
  }
}
